### DISCUSSION TITLE
Working group vs office hour meetings

### DISCUSSION THREAD

**Discussion by yaelharel** [buildpacks/community#115] [Q&A] [answered]
### Background
As far as I remember, we used to have two working group sessions - one on Wednesdays and one on Thursdays.
The one on Wednesdays was the “main” meeting, and many weeks, the number of people on the Thursdays meeting was very small.

Around March 2021, we decided to change the format of the working group meetings on Thursdays and we renamed them to “office hours”.

According to the [office hours document](https://docs.google.com/document/d/1CwixRknj3gAXbi-fTI1-chwtMfy6G3GhdH17aEUjhK0/edit#):
```
The purpose of this meeting is to provide in-depth support and guidance to the community.
Some examples of what questions may be answered in this forum:
* How do I use buildpacks with X?
* Can buildpacks do Y?
* Wouldn't it be cool if buildpacks did Z?
* I'm thinking of submitting this RFC, WDYT?
```

A few weeks ago, in the middle of June 2021, we decided to move the working group meetings to be on Thursdays morning EST.
So now both meetings are on the same day. One in the morning (EST) and one in the afternoon (EST).

In the past few weeks, I've noticed that the office hours were mostly used to continue reviewing RFCs (as we’re doing during most of each the working group meeting).
Due to this change I'm wondering...

### Question
What is the current difference between the working group and the office hour meetings?

### Thoughts
Personally (and also from what I heard from other community members), I don’t think that we should have two meetings to discuss RFCs since most of the discussions should happen on Github. The working group meetings should be used to discuss topics that are harder or takes much longer to have on Github. We expect that after the meeting, the main part of the discussion will be written to the RFC, so people that weren’t able to join the meeting, will be able to review it and be part of it.

I think it would be great if we could use the office hours as a safe forum for people to ask questions, share their thoughts, show some code, etc., as we planned to do when we created it.
It might take some time for people to feel comfortable to do that, but I think it’s worth it.

If we decide to change the format back to two working group meetings every week, it might influence [this PR](https://github.com/buildpacks/community/pull/106) that is discussing the agenda of the working group meetings.

Would love to get your thoughts.
Thanks! 😊

**Comment by natalieparellano** [comment]
> I don’t think that we should have two meetings to discuss RFCs since most of the discussions should happen on Github

I agree very much with this sentiment!

**Comment by cmoulliard** [comment]
> Personally (and also from what I heard from other community members), I don’t think that we should have two meetings to discuss RFCs since most of the discussions should happen on Github.

I fully agree :-)

**Comment by jjbustamante** [comment]
I agree with @yaelharel, because both meetings are in the same day I think we can make them different so people can see the value of joining one or the other and also for keeping the interest of the community and provide the slot of time for people to join and share.

**Comment by natalieparellano** [comment]
I think it's okay if office hours become "whatever the participants for that day want it to be". But I think we should be mindful that whenever there's synchronous discussion happening, people who might still be interested in the topic are inevitably not going to be able to participate. We should aim to capture the salient points from what was discussed in the RFC (preferable I think to notes in the meeting Google doc).

**Comment by jromero** [comment]
To play devil's advocate, the doc also says:

> Some questions may require specific people to be in attendance, because of that **we ask that you add your topic/question to the agenda prior to the meeting.**

I don't think discussing RFCs is an anti-goal of the Office Hours. What we can try to do is prioritize RFC discussions to the bottom of the agenda and leave new members/conversations be first. That said, it does require new members / conversations to be added to the agenda.


EDIT: Note that I'm only talking about topics that go on the agenda and not how conversations about RFCs are "archived". That problem seems seperate to me given that it also applies to WG.

**Comment by yaelharel** [comment, marked as answer]
Thanks to the changes that happened following https://github.com/buildpacks/community/pull/106 and the new core team sync (https://buildpacks.io/community/), I think that the last office hour meetings agenda changed in a good way and their agenda is different than the working group meeting.
I'm going to close this discussion for now.
Thank you for your feedback!
